index.html

```diff
@@ -127,7 +127,7 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
           <li>CSS</li>
           <li>Ruby</li>
         </ul>
-        <button type="button" class="see-project-card" id="button-2">
+        <button type="button" class="see-project-card" id="button-2" onclick="openDetail(2)">
```

Comment:
- [ ] As the previous reviewer suggested,  please ensure you follow the Figma design template as your text on the left of your modal should not be floating onto the image on the right
- [ ] Kindly ensure your list of technologies is to the right as seen in the Template design.
- [ ] Kindly ensured that the text in your button is in centered in the middle of the button.

| Template Design  | Your Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-20 17-22-00](https://user-images.githubusercontent.com/52934128/169570853-239c7815-3908-4e3a-af51-07f620534f9f.png)  | ![Screenshot from 2022-05-20 17-17-08](https://user-images.githubusercontent.com/52934128/169571335-6a549366-2907-4974-988e-4fc5f88bc9e4.png)  |

Reply:
@topeogunleye Thanks for your review, I am going to change the buttons and the text style. 

However, the technologies is on top of the image. **This template design is not the popUp model.**

Reply:
Okay. That's great. :+1: , sorry for using the wrong template design